syntax/parse.go

```diff
@@ -16,15 +16,21 @@ import "log"
 // Enable this flag to print the token stream and log.Fatal on the first error.
 const debug = false
 
+type Mode uint
+
+const (
+	RetainComments Mode = 1 << iota
```

Comment:
RetainComments Mode = 1 << iota // retain comments in AST; see Node.Comments

Reply:
Done